doc/collision_detection.md

```diff
@@ -166,8 +166,7 @@ A Shape is the base class for representing a scalable geometrical shape. The sha
 ways of defining how they look, but they all have a size and angle that can be modified and the
 shape definition will scale or rotate the shape accordingly.
 
-There are currently three shapes: [Polygon](#Polygon), [Rectangle](#Rectangle) and
-[Cirlce](#Circle).
+There are currently three shapes: [](#polygon), [](#rectangle) and [](#circle).
```

Comment:
Shouldn't there still be text in []?

Reply:
Turns out it's optional. I was just testing here that that is indeed so.

Reply:
But then it has the # before right?

Reply:
No, the `#` is only for linking. The actual title will come from the text of the header itself, which in this case is `Polygon`.

Reply:
Oh, that's great!